apps/opik-backend/src/main/java/com/comet/opik/domain/DatasetItemResultMapper.java

```diff
@@ -71,26 +72,6 @@ static JsonNode getJsonNodeOrNull(Object field) {
         return JsonUtils.getJsonNodeFromString(field.toString());
     }
 
-    private static List<FeedbackScore> getFeedbackScores(Object feedbackScoresRaw) {
```

Comment:
This code was moved to a mapper to simplify the dao class and make it reusable. Why do you need this back in the DAO class? Can you modify or reuse the method in the mapper?